htdocs/mri/jiv/get_file.php

```diff
@@ -80,76 +73,65 @@
 }
 unset($path_parts);
 
-// Make sure that the user isn't trying to break out of the $path by
-// using a relative filename.
-// No need to check for '/' since all downloads are relative to $imagePath,
-// $DownloadPath or $mincPath
-if (strpos("..", $File) !== false) {
-    error_log("ERROR: Invalid filename");
-    header("HTTP/1.1 400 Bad Request");
-    exit(4);
-}
```

Comment:
I don't foresee any problems with `realpath()`. I just see benefits of defence in depth over a single point of failure.